### ISSUE TITLE
Won't start with error - Cannot read property 'time' of null

### ISSUE THREAD

**Issue by Mikadily** [michaelgrosner/tribeca#29]
I'm having trouble following docker installation guide. It works fine for bitfinex, but not okcoin or hitbtc (didn't try coinbase).

My sample Dockerfile:
```
FROM node:0.12.7
RUN apt-get update

RUN apt-get install -y git

RUN git clone https://github.com/michaelgrosner/tribeca.git

WORKDIR tribeca

RUN npm install -g grunt-cli tsd forever
RUN npm install

RUN tsd reinstall -s
RUN grunt compile

EXPOSE 3000 5000

# General config properties. Properties with `NULL` should be replaced with your own exchange account information.
ENV TRIBECA_MODE prod
ENV EXCHANGE hitbtc
ENV TradedPair BTC/USD
ENV WebClientUsername NULL
ENV WebClientPassword NULL
ENV WebClientListenPort 3000
# IP to access mongo instance. If you are on a mac, run `boot2docker ip` and replace `tribeca-mongo`.
ENV MongoDbUrl mongodb://tribeca-mongo:27017/tribeca

# DEV
## HitBtc
ENV HitBtcPullUrl http://api.hitbtc.com
ENV HitBtcOrderEntryUrl wss://api.hitbtc.com:8080
ENV HitBtcMarketDataUrl ws://api.hitbtc.com:80
ENV HitBtcSocketIoUrl https://api.hitbtc.com:8081
ENV HitBtcApiKey xxx
ENV HitBtcSecret xxx
ENV HitBtcOrderDestination HitBtc

WORKDIR tribeca/service
CMD ["forever", "main.js"]
```

And error I get:
```
info: 9/2/15 15:35:28,406       [tribeca:mtbroker]      loaded 0 market trades
error: Terminating! uncaughtException [TypeError: Cannot read property 'time' of null] TypeError: Cannot read property 'time' of null
    at LoaderSaver.saver (/tribeca/tribeca/service/persister.js:35:25)
    at Persister.MarketTradesLoaderSaver.saver [as _saver] (/tribeca/tribeca/service/markettrades.js:20:32)
    at Persister.persist (/tribeca/tribeca/service/persister.js:139:19)
    at EventEmitter.MarketTradeBroker.handleNewMarketTrade (/tribeca/tribeca/service/markettrades.js:58:30)
    at EventEmitter.emit (events.js:107:17)
    at Evt.trigger (/tribeca/tribeca/service/utils.js:41:62)
    at /tribeca/tribeca/service/gateways/hitbtc.js:109:35
    at Array.forEach (native)
    at Request._callback (/tribeca/tribeca/service/gateways/hitbtc.js:105:42)
    at Request.self.callback (/tribeca/node_modules/request/request.js:198:22)
```

What gives?

**Comment by Mikadily** [comment]
I've tested bitstamp and btce as well. All with different errors (bitstamp almost works). So perhaps it's just API problem.

**Comment by michaelgrosner** [comment]
I fixed the issue in 10b86b816e44bbf780bf214f849d623b6ba84c6a -- can happen when there is a trade in the market without a quote generated yet by the code

**Comment by Mikadily** [comment]
This solved the issue. Now there are different one's (I'll raise another issue), but at least it doesn't crash.

Thanks!
